src/main/java/org/radarbase/appserver/exception/handler/ResponseEntityExceptionHandler.java

```diff
@@ -59,11 +59,31 @@ public final ResponseEntity<ErrorDetails> handleInvalidNotificationDetailsExcept
         return handleEntityWithoutCause(ex, request);
     }
 
+    @ExceptionHandler(MessageTransmitException.class)
+    public final ResponseEntity<ErrorDetails> handleMessageTransmitException(Exception ex, WebRequest request) {
+        return handleEntityWithCause(ex, request);
+    }
```

Comment:
can be removed as don't need a response for these exceptions

Reply:
Removed!